bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ChaumPedersen.cs

```diff
@@ -192,6 +192,33 @@ unsafe internal DisjunctiveChaumPedersenProof(NativeDisjunctiveChaumPedersenProo
         /// <Summary>
         /// make function for a `DisjunctiveChaumPedersenProof`
         ///
+        /// This overload does not accept a seed value and calculates
+        /// proofs independent of the original encryption. (faster performance)
+        /// <param name="message"> The ciphertext message</param>
+        /// <param name="r"> The nonce used creating the ElGamal ciphertext</param>
+        /// <param name="k"> The public key of the election</param>
+        /// <param name="q"> A value used when generating the challenge,
+        ///          usually the election extended base hash (𝑄')</param>
+        /// <param name="plaintext">The constant value to prove, zero or one</param>
+        /// <returns>A unique pointer</returns>
+        /// </Summary>
+        unsafe public DisjunctiveChaumPedersenProof(
+            ElGamalCiphertext message,
+            ElementModQ r,
```

Comment:
Are these variable names meaningful?  If yes, ignore this comment.  Else would it be worthwhile to improve them in both C# and C++ to what's in the descriptions eg `nonce`, `publicKey`, `extendedBaseHash`?  This comment isn't specific to DisjunctiveChaumPedersenProof, it can be extended to places like NativeInterface.

Reply:
where single letter variables are used they are used to remain consistent with python. i go back and forth on wether or not we should change them. on one hand they are useful in this form since they map directly to the specification, on the other hand they are not descriptive at all for an engineer simply trying to integrate the SDK.